src/components/data/DataPathLink.js

```diff
@@ -0,0 +1,82 @@
+import { CircularProgress, Typography } from "@material-ui/core";
+import {
+    getFolderPage,
+    getParentPath,
+    parseNameFromPath,
+} from "components/data/utils";
+import { ERROR_CODES, getErrorCode } from "components/error/errorCode";
+import ErrorTypographyWithDialog from "components/error/ErrorTypographyWithDialog";
+import AppParamTypes from "components/models/AppParamTypes";
+import DELink from "components/utils/DELink";
+import { useTranslation } from "i18n";
+import Link from "next/link";
+import React from "react";
+import { useDataDetails } from "serviceFacades/filesystem";
+
+export default function DataPathLink(props) {
+    const { id, param_type, path } = props;
+
+    const [statError, setStatError] = React.useState();
+    const [otherError, setOtherError] = React.useState();
+
+    React.useEffect(() => {
+        if (!path) {
+            setStatError(null);
+            setOtherError(null);
+        }
+    }, [path]);
+
+    const displayValue = parseNameFromPath(path);
+    const linkTarget = getFolderPage(
+        AppParamTypes.FOLDER_INPUT === param_type ? path : getParentPath(path)
+    );
+
+    const { t } = useTranslation("analyses");
+
+    const { isFetching: isFetchingStat } = useDataDetails({
+        paths: [path],
+        enabled: !!path,
+        onSuccess: () => {
+            setStatError(null);
+            setOtherError(null);
+        },
+        onError: (error) => {
+            if (ERROR_CODES.ERR_DOES_NOT_EXIST === getErrorCode(error)) {
+                setStatError(t("errorInputDoesNotExist", { path: linkTarget }));
```

Comment:
Minor, from the screenshot this path in the error message seems to include the prefix `/data/ds`.  I wonder if that might be confusing.  It's sort of a blend between the DE URL and the path in iRods, but not completely either.

Reply:
Agreed. It should be just the path.